using-mobile-extensions/mobile-core/configuration/README.md

```diff
@@ -0,0 +1,65 @@
+# Configuration
+
+The Configuration extension is built into the Mobile Core.  With its APIs you will generate the necessary events in the Event Hub that will provide configuration options for all installed extensions.
+
+## Environment-aware configuration properties
+{% hint style="info" %}
+This feature is only available in iOS ACPCore v2.0.3 or newer
+{% endhint %}
+
+Some extension developers will choose to use different configuration values based on the working environment.  In that case, the configuration generated may have several entries for the same property.  For example, the Adobe Campaign Standard extension has different endpoints for development, staging, and production servers.  An example of what a raw configuration supporting multiple build environments might look like is shown below:
```

Comment:
Please change:

- "Some extension developers will choose" to "Some extension developers might use"
- change "In that case" to "In this case"
- change "may" to "might"
- Change "An example of what a raw configuration supporting an example of what a raw configuration supporting multiple build environments might look like is shown below:" to "Here is an example of a raw configuration that supports multiple build environments:"